src/browser/request.ts

```diff
@@ -106,7 +106,7 @@ async function startFetchRequest(
     () => controller?.abort(),
   );
   const headers: { [key: string]: any } = {};
-  // @ts-expect-error
+  // @ts-expect-error no .keys()?
   for (const headerName of res.headers.keys()) {
```

Comment:
I added this `ts-expect-error` when updating TS to include DOM types.
It's probably broken (unless TS types are wrong) but I left it as it is because no other code users this (even in e2e).

Reply:
and I just did my part of adding why we need the `expect-error` 🤷‍♂️